### PR TITLE
Apim 3926 add markdown

### PR DESCRIPTION
**Pull request by jourdiw**
## Issue

https://gravitee.atlassian.net/browse/APIM-3926

## Description

Create a page markdown component to be used for documentation rendering

https://github.com/gravitee-io/gravitee-api-management/assets/42294616/d7fdc7c3-9df7-43f1-9841-e3b80b8ea59d

Next PR: Integrate attached media for markdown pages

### CODE REVIEW

File: gravitee-apim-portal-webui-next/src/components/page/page.component.ts [hunk 1]
```diff
@@ -15,17 +15,22 @@
...
 @Component({
   selector: 'app-page',
   standalone: true,
-  imports: [PageSwaggerComponent],
+  imports: [PageSwaggerComponent, PageMarkdownComponent],
   templateUrl: './page.component.html',
   styleUrl: './page.component.scss',
 })
 export class PageComponent {
   @Input()
```
Comment (jansobczynski) [inline gravitee-apim-portal-webui-next/src/components/page/page.component.ts:30]: As I understand: pages is an array of objects and page is a single object. So are we passing the same objects in both inputs?
It highlights another line, but I meant lines 30-34

Comment (jourdiw) [reply gravitee-apim-portal-webui-next/src/components/page/page.component.ts:30]: `pages` is used for the list of pages of a given API. `page` is the current page shown.  The current page being shown is in the list of pages, but I find it simpler to just have a clear input of which page to show with `page`. 

Does that make sense? Would it help to rename the variables?

Comment (jansobczynski) [reply gravitee-apim-portal-webui-next/src/components/page/page.component.ts:30]: It make sense now, renaming the variables would be good, thanks!

Comment (jourdiw) [reply gravitee-apim-portal-webui-next/src/components/page/page.component.ts:30]: Done

**Review by jansobczynski — APPROVED** [review]

**Pull request merged**